cannot compile serde_json 1.0.45

```
   Compiling serde_json v1.0.45
error[E0405]: cannot find trait `StdError` in module `serde::de`
   --> /Users/xin/.cargo/registry/src/github.com-1ecc6299db9ec823/serde_json-1.0.45/src/error.rs:317:17
    |
317 | impl serde::de::StdError for Error {
    |                 ^^^^^^^^ not found in `serde::de`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0405`.
error: could not compile `serde_json`.
```

toolchain 
```
active toolchain
----------------

stable-x86_64-apple-darwin (default)
rustc 1.40.0 (73528e339 2019-12-16)
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -17,7 +17,7 @@ travis-ci = { repository = "serde-rs/json" }
 appveyor = { repository = "serde-rs/json" }
 
 [dependencies]
-serde = { version = "1.0.60", default-features = false }
+serde = { version = "1.0.100", default-features = false }
 indexmap = { version = "1.2", optional = true }
 itoa = { version = "0.4.3", default-features = false }
 ryu = "1.0"

